code/parse/sexp.cpp

```diff
@@ -15619,9 +15555,9 @@ int sexp_path_flown()
 	return SEXP_FALSE;
 }
 
-void sexp_send_training_message(int node)
+void sexp_send_training_message(const int node)
```

Comment:
Why is this the only time `const` was added?

Reply:
I don't actually remember. :man_shrugging: I've removed it.